tests/testthat/test-ard_total_n.R

```diff
@@ -16,3 +16,30 @@ test_that("ard_total_n() follows ard structure", {
       check_ard_structure(method = FALSE)
   )
 })
+
+test_that("ard_total_n() errors with incomplete factor columns", {
```

Comment:
Do we need these tests, since this function doesn't operate on any columns?

Reply:
Factor checks removed from the `ard_total_n()` tests